internal/controller/controllers/images.go

```diff
@@ -0,0 +1,48 @@
+/*
+Copyright 2021.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package controllers
+
+import (
+	"os"
+)
+
+func ServiceImage() string {
+	return getEnvVar("SERVICE_IMAGE", "quay.io/ocpmetal/assisted-service:latest")
```

Comment:
Could/should these be set in the CR?

Reply:
Typically with operators we want the specific images to be defined with the operator packaging (CSV). That way mirroring of images is done reliably, and upgrade is handled by OLM in a clearly-defined way.

From a user/customer perspective, they know that if they have XYZ operator v1.2.3 running, that corresponds to specific artifacts built by us, and we can reliably reason about bugs and security fixes.